src/inc/eventtracebase.h

```diff
@@ -374,7 +469,9 @@ namespace ETW
         VOID Append(SIZE_T currentFrame);
         EtwStackWalkStatus SaveCurrentStack(int skipTopNFrames=1);
     public:
+#if !defined(FEATURE_PAL)
         static ULONG SendStackTrace(MCGEN_TRACE_CONTEXT TraceContext, PCEVENT_DESCRIPTOR Descriptor, LPCGUID EventGuid);
+#endif //!defined(FEATURE_PAL)
         EtwStackWalkStatus GetCurrentThreadsCallStack(UINT32 *frameCount, PVOID **Stack);
```

Comment:
I have only put code which does not compile under non windows platform under !defined(FEATURE_PAL), this function compiles fine but is not used. I did not look at such functions should i ?